buildSrc/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -172,16 +172,16 @@ normalization {
 }
 
 fun isJavaVersionAllowed(version: JavaVersion): Boolean {
```

Comment:
Can this whole method be encapsulated into `OtelJavaExtension`?

Reply:
I guess it could - not sure there's a real reason to unless we can see ourselves calling this from the projects themselves. Here, it would just cause the below invocations to become more verbose, `otelJava.isJavaVersionAllowed` so don't think it's a win.